integrations-and-supported-tools/fastai/scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fastai==2.3.1
+neptune-client==0.9.16
```

Comment:
Could you test on newest neptune?

Reply:
Sure, we can.

Actually that's something Rafal and I are trying to agree, because fastai didn't have save_every_n_epoch capability until 2 days ago. So now we have to rethink our implementation.